portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -205,7 +205,12 @@ definition {
 			callbackURI = '''http://martamedio.com/oauth2-tester/?url=${portalURL}&client_id=abc123''',
 			clientId = "abc123",
 			clientProfile = "Web Application",
-			clientSecret = "abc123");
+			clientSecret = "abc123",
+			customPortalURL = "true");
```

Comment:
Could you please refactor the test case / macro, to eliminate this new custom property?

Reply:
We could trick baseURL-»portalURL and refactor the OAuth2.addAndEditExternalApplication macro, to use portal.url only, when we don't define any value at test case level.